Source/JavaScriptCore/API/tests/testapi.c

```diff
@@ -1382,6 +1382,54 @@ static void testCFStrings(void)
 }
 #endif
 
+static bool samplingProfilerTest(void)
+{
+#if ENABLE(SAMPLING_PROFILER)
+    JSContextGroupRef contextGroup = JSContextGroupCreate();
+    JSGlobalContextRef context = JSGlobalContextCreateInGroup(contextGroup, NULL);
+    {
+        bool result = JSContextGroupEnableSamplingProfiler(contextGroup);
+        if (result)
+            printf("PASS: Enabled sampling profiler.\n");
+        else {
+            printf("FAIL: Failed to enable sampling profiler.\n");
+            return false;
```

Comment:
Here, `samplingProfilerTest` returning `false` means it fails.  This does not match the caller's expectation.

I know the precedence is to use true / false.  Maybe you should introduce in this file a PASS / FAIL enum / macro that maps to false / true (matching current convention) to make the code easier to read?

Reply:
Actually, I feel like still using false / true would be good here, largely due to the fact that this is C file (testapi.c).
We can make it enum, but I would like to defer since this is out of scope of this patch.